sklearn/mixture/base.py

```diff
@@ -172,11 +172,18 @@ def _initialize(self, X, resp):
     def fit(self, X, y=None):
         """Estimate model parameters with the EM algorithm.
 
-        The method fit the model `n_init` times and set the parameters with
+        The method fits the model `n_init` times and sets the parameters with
         which the model has the largest likelihood or lower bound. Within each
         trial, the method iterates between E-step and M-step for `max_iter`
         times until the change of likelihood or lower bound is less than
         `tol`, otherwise, a `ConvergenceWarning` is raised.
+        If `warm_start` is `True`, then `n_init` is ignored and a single
```

Comment:
should be double backticks everywhere....

Reply:
Oh right, just fixed this, thanks.